roles/pulp_repository/defaults/main.yml

```diff
@@ -12,6 +12,14 @@ pulp_repository_rpm_repos: []
 pulp_repository_sync_retries: 3
```

Comment:
Naming is a little off here. Would suggest either renaming this to `pulp_repository_retries` or adding separate defaults for repositories and remotes.

Reply:
Good call